privacyPolicy.txt

```diff
@@ -0,0 +1,29 @@
+This plugin can be configured to collect usage data that is sent to Microsoft to help improve the product.
+
+If you do not want this information reported you can:
+- Click 'Decline' when initially prompted on startup whether you are willing to allow usage data to be collected.

```

Comment:
Privacy requested edit - Change text to:
    "Click 'Decline' when initially prompted on startup to stop usage data from being collected."
The existing text is confusing.